### ISSUE TITLE
Gravatar.com contacted even when disabled or with disableThirdPartyRequests 

### ISSUE THREAD

**Issue by rasos** [jitsi/jitsi-meet#14107]
We still still see Gravatar.com being contacted, even when disabling it or replacing it with libravatar. 

Using Dockerized stable-9078 with config: 

```
// Setup for Gravatar-compatible services.
// Defaults to gravatar service - which is even contacted when disabled below, so we redirect to a libravatar server - but it seems not to connect there
config.gravatar.baseUrl = 'https://www.libravatar.org/avatar/';
// True if Gravatar should be disabled.
config.gravatar.disabled = true;
// disabling seems not to work either
config.disableThirdPartyRequests = true;
```
I still see in the network console:

> https://www.gravatar.com/avatar/45da90b185246e02402c14efe2639ad0?d=404&size=200 

This privacy issue has been already tackled with https://github.com/jitsi/jitsi-meet/pull/9300 and https://github.com/jitsi/jitsi-meet/issues/5670 . 

It seems to happen only, if an e-mail address has been made available in the profile through a JWT token. We have a dev instance without JWT which does follow the disable settings as expected.

**Comment by rasos** [comment]
Ok, I saw in the console a 

> Uncaught TypeError: config.gravatar is undefined

so I added in the config

```
if (!config.hasOwnProperty('gravatar')) config.gravatar = {};
config.gravatar.disabled = true;
```

Now Gravatar.com is not contacted anymore and if enabled I can even use any Gravatar-compatible service.

This is independent of setting `config.disableThirdPartyRequests = true;` - I had assumed that this would also restrict any Gravatar service. But as Gravatar can be disabled separately, all is fine. 

So I learned that mapping configs, which are not covered by docker env, need special  attention and a careful syntax check.
